data/sea/54-zebra.h

```diff
@@ -206,22 +270,42 @@ void zebra_collect_state (zebra_config_t *cfg, zebra_state_t *state)
     free (state);
```

Comment:
don't need to free the buffer, entity_fact_offset etc? (I don't think this is important)